jetstream/outcomes/firefox_desktop/networking.toml

```diff
@@ -67,3 +84,291 @@ log_space = true
+
+[metrics.tls_successful_connections_overall.statistics.kernel_density_estimate]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_connections_overall.statistics.empirical_cdf]
+pre_treatments = ['remove_nulls']
+
+
+[metrics.tls_successful_connections_with_ech]
+data_source = "main"
+select_expression = '{{agg_hist_zero_fract("payload.histograms.ssl_handshake_result_ech")}}'
+type = "scalar"
+
+[metrics.tls_successful_connections_with_ech.statistics.bootstrap_mean]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_connections_with_ech.statistics.deciles]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_connections_with_ech.statistics.kernel_density_estimate]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_connections_with_ech.statistics.empirical_cdf]
+pre_treatments = ['remove_nulls']
+
+
+[metrics.tls_successful_connections_with_ech_grease]
+data_source = "main"
+select_expression = '{{agg_hist_zero_fract("payload.histograms.ssl_handshake_result_ech_grease")}}'
+type = "scalar"
+
+[metrics.tls_successful_connections_with_ech_grease.statistics.bootstrap_mean]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_connections_with_ech_grease.statistics.deciles]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_connections_with_ech_grease.statistics.kernel_density_estimate]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_connections_with_ech_grease.statistics.empirical_cdf]
+pre_treatments = ['remove_nulls']
+
+
+[metrics.tls_successful_first_try_connections]
+data_source = "main"
+select_expression = '{{agg_hist_zero_fract("payload.histograms.ssl_handshake_result_first_try")}}'
+type = "scalar"
+
+[metrics.tls_successful_first_try_connections.statistics.bootstrap_mean]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_first_try_connections.statistics.deciles]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_first_try_connections.statistics.kernel_density_estimate]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_first_try_connections.statistics.empirical_cdf]
+pre_treatments = ['remove_nulls']
+
+
+[metrics.tls_successful_conservative_connections]
+data_source = "main"
+select_expression = '{{agg_hist_zero_fract("payload.histograms.ssl_handshake_result_conservative")}}'
+type = "scalar"
+
+[metrics.tls_successful_conservative_connections.statistics.bootstrap_mean]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_conservative_connections.statistics.deciles]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_conservative_connections.statistics.kernel_density_estimate]
+pre_treatments = ['remove_nulls']
+
+[metrics.tls_successful_conservative_connections.statistics.empirical_cdf]
+pre_treatments = ['remove_nulls']
+
+
+[metrics.quic_successful_connections_no_ech]
+data_source = "main"
+select_expression = "{{agg_hist_zero_fract(mozfun.map.get_key(payload.histograms.http3_ech_outcome, 'NONE'))}}"
```

Comment:
There are some `"` missing. `agg_hist_zero_fract` expects a string parameter which represents a SQL snippet.

```suggestion
select_expression = """{{agg_hist_zero_fract("mozfun.map.get_key(payload.histograms.http3_ech_outcome, 'NONE')")}}"""
```

There are a couple more of these calls below which need to be fixed as well.